server/data/nl/danielsjf.txt

```diff
@@ -2871,3 +2871,507 @@ Onze droger is alweer stuk.
+Ik heb mijn vettige handen met zijn zeep gewassen.
+Hij teelde al jarenlang dezelfde gewassen.
+De butler serveerde het eten. 
+Hun bedrijf bevindt zich in een grijze zone.
+Niemand weet goed wat ze gaan doen met de weeskinderen.
+Er is een wet gestemd vanmorgen.
+Zij zetelt in het parlement.
+De regering is met spoed samengekomen. 
+Hij had argwaan en besloot om het contract niet te ondertekenen.
+Mag ik je handtekening?
+Voor hun huis zijn ze een lening van twintig jaar aangegaan.
+Het boek was vlot geschreven.
+Zij kon altijd goede verhalen vertellen.
+Het was een spannende achtervolging.
+Ik ben tegen de doodstraf.
+Zij is haar land moeten ontvluchten omdat de vervolgt werd voor politieke misdaden. 
+We naderen het einde.
+Er was bijna geen volk in het pretpark dus mochten we tweemaal na elkaar de rit doen.
+Op Pinksteren hebben we een dag vrijaf.
+Ik heb geen bezwaren.
+In het centrum staat een prachtige kathedraal.
+Je mag nu niet dralen.
+Ja, we gaan dat programma hervatten.
+Nee, dat is niet meer het geval.
+Ik heb mijn excuses aangeboden.
+Je moet je excuseren voor dat voorval.
+Is dat een veilige fabriek?
+Ik had geen andere keuze.
+Ze was verontwaardigd.
+Ze toverde een glimlach op haar gezicht.
+Hij was aangedaan door het verlies.
+Mijn neef kent iets van auto's.
+Mijn nicht kan je daar bij helpen.
+Ik heb iemands leven gered door hem te reanimeren.
+De papieren waren vergeeld door de jaren.
+Een slechte verbranding kan koolstofmonoxide creëren.
+De brandstofpomp was stuk.
+Ze was onoverwinnelijk. 
+Eenmaal bezig is ze niet meer te stoppen.
+Soms voelde hij zich een vreemde in zijn eigen land.
+Hij had het stopbord niet gezien en reed zonder te kijken het kruispunt op.
+Ze verleende voorrang aan de vrachtwagen.
+Hij stak over aan het zebrapad.
+Ze kon nog net de middenberm ontwijken.
+De jager schoot maar miste de fazant.
+Het jachtseizoen is weer begonnen.
+Zijn tegenstander was veel sterker en hij maakte geen schijn van kans.
+Hij kon zijn schoonbroer niet luchten.
+Ze kwam goed overeen met haar schoonbroer.
+Ze probeerde hem een hart onder de riem te steken.
+Zijn schoonzus kwam iedere zondag op bezoek.
+Wacht, ik zal je een diagram tekenen.
+Hij geloofde haar voor geen meter.
+Sinds jaar en dag was hij atheïst.
+Haar ouders hebben haar teveel verwend.
+Hij had slechte bedoelingen.
+Door de sterke windvlagen kwam hij nauwelijks vooruit op zijn fiets.
+Ze is ooit nog de landskampioen geweest.
+Ze heeft een valse lach.
+Er ligt nog een oude krant in de schuif.
+Steek je de haard aan?
+Negen nette neven negeren vier vieze venten.
+We moeten concluderen dat onze pogingen gefaald zijn.
+Er komt een dodelijke asteroïde op ons af.
+Dit betekent het einde van de mensheid.
+Bunkers kunnen ons niet beschermen
+Tsunami’s zullen de kustzones treffen.
+De temperatuur zal enorm toenemen.
+Het team speelde goed samen.
+Hockey is een populaire sport.
+Er is een groot verschil tussen hockey en ijshockey.
+De resultaten waren niet navenant.
```

Comment:
is "navenant" het juiste woord hier? zou er ander "kwamen niet overeen" van maken 
Ik kende het woord niet en google zegt dat het hoort bij een vergelijking, maar dat is hier niet het geval